src/cmd-build

```diff
@@ -61,6 +65,23 @@ prepare_build
 ostree --version
 rpm-ostree --version
 
+# RFC3339 formatter
+RFC3339="%Y-%m-%dT%H:%M:%SZ"
+
+if [ "${TIMESTAMP}" == 1 ]; then
+    if [ ! -e "${workdir:?}"/builds/builds.json ]; then
+        fatal "No builds.json file found; unable to apply timestamp"
+    fi
+    builds_json="${workdir}"/builds/builds.json
+    tmp_json="${workdir}"/builds/tmp.json
+    timestamp=$(date -u +$RFC3339)
+    echo "New timestamp: ${timestamp}"
+    jq --arg TS "${timestamp}" '.timestamp = $TS' "${builds_json}" > "${tmp_json}"
```

Comment:
You control the format string, so you know the timestamp will not contain quotes.  That means you could use:

```sh
jq ".timestamp = \"${timestamp}\"" "${builds_json}" > "${tmp_json}"
```

here.  I'm on the fence about which is more readable, with my alternative having fewer options (yay!) but escaped quotes (boo!).  Feel free to keep what you have (obviously, since I'm just the peanut gallery here ;).

Reply:
We welcome comments from the peanut gallery!  :laughing: 

Thanks for the suggestions; I'll incorporate them in the next commit.